src/capplet/mate-notification-applet.c

```diff
@@ -173,16 +173,19 @@ applet_main (MatePanelApplet *applet_widget)
   textdomain (GETTEXT_PACKAGE);
 #endif /* ENABLE_NLS */
 
+#ifndef ENABLE_IN_PROCESS
   g_set_application_name (_("Do Not Disturb"));
+#endif
   gtk_window_set_default_icon_name ("mate-notification-properties");
 
   applet = g_new (MateNotificationApplet, 1);
   applet->applet = applet_widget;
   applet->settings = g_settings_new (GSETTINGS_SCHEMA);
 
+#ifndef ENABLE_IN_PROCESS
   /* needed to clamp ourselves to the panel size */
   mate_panel_applet_set_flags (MATE_PANEL_APPLET (applet), MATE_PANEL_APPLET_EXPAND_MINOR);
-
+#endif
```

Comment:
Same question here?

Reply:
Another example of using
`ifndef` not `ifdef`
to keep a line of code to out of process only. This one too throws a warning if run in-process

Reply:
sorry for that noise